src/encoded/types/target.py

```diff
@@ -54,6 +54,8 @@ def organism(self, properties=None, return_uuid=False):
                 )
                 raise ValidationFailure('body', ['genes'], msg)
             organism_uuid = next(iter(organism_uuids))
+        else:
+            return None
```

Comment:
This doesn't show up in JSON object as `organism: None` right?

Reply:
Correct. Won't have the "organism" calculated property at all.